web/src/beta/utils/value.ts

```diff
@@ -175,14 +180,14 @@ export function valueToGQL<T extends ValueType>(
   type: T
 ) {
   if (type === "camera" && val && typeof val === "object" && "height" in val) {
-    return {
-      ...(val as Camera),
-      altitude: (val as Camera).height
-    };
+    const { height, aspectRatio, ...rest } = val as Camera;
+    const v: DataCamera = { ...rest, altitude: height }; 
+    return v;
```

Comment:
Could you refactor a little bit to make sure aspectRatio been excluded even there's no `height` in val?